core/src/it/scala/org/ensime/core/JavaCompilerSpec.scala

```diff
@@ -390,7 +390,8 @@ class JavaCompilerSpec extends EnsimeSpec with OptionValues
       }
     }
   }
-  it should "support Java 8 JavaFx addListener features" in {
+
+  it should "support Java 8 JavaFx addListener lambda features" in {
```

Comment:
Yes, it's a general lambda problem. When there is only `T<? extends T>`, no lambda, it works. But, if this is in a lambda with wildcard `(T<>)`, it fails.